src/main/java/controller/LottoGameController.java

```diff
@@ -42,13 +45,27 @@ private LottoMoney getLottoMoney() {
 		}
 	}
 
+	private Lottos buyLottos(LottoMoney lottoMoney) {
+		int manualLottoCount = lottoGameView.getManualLottoCount();
+		LottoMoney remain = lottoMoney.calculateRemainAfterBuy(manualLottoCount);
+
+		List<Lotto> manualLottos = getManualLottos(manualLottoCount);
+		List<Lotto> autoLottos = lottoGenerator.generateLottos(remain.calculateLottoCount());
```

Comment:
사소해보일 수 있지만 변수를 선언한 위치와 사용하는 위치를 최대한 가깝게 위치시켜보면 어떨까요? 😉

```suggestion
		int manualLottoCount = lottoGameView.getManualLottoCount();
		List<Lotto> manualLottos = getManualLottos(manualLottoCount);
		
		LottoMoney remain = lottoMoney.calculateRemainAfterBuy(manualLottoCount););
		List<Lotto> autoLottos = lottoGenerator.generateLottos(remain.calculateLottoCount());
```

[클린 코드 - 수직거리](https://github.com/Yooii-Studios/Clean-Code/blob/master/Chapter%2005%20-%20%ED%98%95%EC%8B%9D%20%EB%A7%9E%EC%B6%94%EA%B8%B0.md#%EB%B3%80%EC%88%98%EC%84%A0%EC%96%B8)를 참고해보시면 좋을 것 같아요. 🙂